src/bp-templates/bp-nouveau/buddypress/media/albums.php

```diff
@@ -10,7 +10,13 @@
 	<div class="bb-media-actions-wrap album-actions-wrap">
 		<h2 class="bb-title"><?php esc_html_e( 'Albums', 'buddyboss' ); ?></h2>
 		<?php
-		if ( bp_is_profile_albums_support_enabled() ) {
+		if ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) {
```

Comment:
@mahedihasannoman Here we can use single if statement with OR. As a content statement for if and else is same.

Reply:
Done